package.json

```diff
@@ -1,30 +1,38 @@
 {
     "name": "supertype",
     "description": "A type system for classical inheritence, mix-ins and composition.",
+    "version": "3.0.0",
+    "main": "dist/index.js",
+    "types": "dist/index.d.ts",
     "homepage": "https://github.com/haven-life/supertype",
-    "version": "2.2.7",
     "dependencies": {
-        "reflect-metadata": "^0.1.9"
+        "@types/reflect-metadata": "^0.1.0",
+        "reflect-metadata": "^0.1.12"
     },
     "devDependencies": {
+        "@types/chai": "^3.4.34",
+        "@types/mocha": "^2.2.39",
+        "@types/node": "^7.0.5",
         "chai": "*",
         "eslint": "3.7.x",
         "istanbul": "0.4.5",
+        "q": "1.x",
+        "sinon": "1.15.x",
+        "typescript": "^2.8.4",
```

Comment:
should lock to specific version of typescript (same with all modules)

Reply:
locking it to 2.9.2